test/reader_test.exs

```diff
@@ -157,16 +157,16 @@ defmodule ExsodaTest.Reader do
     result = Enum.into(stream, [])
 
     assert result == [
-      [{"Region", "0km SE of Sakai, Japan"}, {"Magnitude", "4.6"}],
```

Comment:
Why the casing change? Is this unrelated to the connection pool stuff?

Reply:
yea, the tests started failing. I'm assuming because NBE/OBE stuff?